syntax = "proto3";


service LedSign {
  rpc HealthCheck (HealthCheckRequest) returns (HealthCheckResponse);
  rpc UpdateSignText (LedSignRequest) returns (LedSignResponse);
}

message HealthCheckRequest{
  string officer_name = 1;
}

message HealthCheckResponse{
  string message = 1;
}

message LedSignRequest{
  string text = 1;
  int32 brightness = 2;
  int32 scroll_speed = 3;
  string background_color = 4;
  string text_color = 5;
  string border_color = 6;
}

message LedSignResponse{
    string message = 1;
}
